packages/atlas-core/package.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "atlas-core",
   "moduleName": "Atlas Core",
-  "version": "3.9.3",
+  "version": "3.10.0",
   "license": "Apache-2.0",
   "copyright": "© Mendix Technology BV 2022. All rights reserved.",
   "repository": {
     "type": "git",
     "url": "https://github.com/mendix/atlas.git"
   },
   "marketplace": {
-    "minimumMXVersion": "9.18.4.62522",
+    "minimumMXVersion": "9.24.0.2965",
     "marketplaceId": 117187
   },
   "testProject": {
     "githubUrl": "https://github.com/mendix/Atlas-Design-System",
-    "branchName": "lts/9.18"
+    "branchName": "main"
```

Comment:
Should we maintain the same branch naming for this new lts version instead of main?

Reply:
We don't need it right now as we will split packages. Later, after split, we can do proper branching.